.env.example

```
@@ -3,13 +3,23 @@
 # ===========================================
 # Copy this file to .env and fill in your actual values
 
+ENVIRONMENT=development
+DEBUG_MODE=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEBUG_MODE key should go before the ENVIRONMENT key

(UnorderedKey)